examples/step-70/step-70.cc

```diff
@@ -0,0 +1,1238 @@
+    int          max_level_refinement = 5;
+    std::string  refinement_strategy  = "fixed_fraction";
+    double       coarsening_fraction  = 0.3;
+    double       refinement_fraction  = 0.3;
+    unsigned int max_cells            = 1000;
+    int          mod_refinement       = 5;
+    int          mod_output           = 1;
+
+    mutable ParameterAcceptorProxy<Functions::ParsedFunction<spacedim>> rhs;
+    mutable ParameterAcceptorProxy<Functions::ParsedFunction<spacedim>>
+      angular_velocity;
+  }; // namespace Step70
+
+
+  template <int spacedim>
+  class SolidVelocity : public Function<spacedim>
+  {
+  public:
+    SolidVelocity(const Functions::ParsedFunction<spacedim> &angular_velocity)
+      : angular_velocity(angular_velocity)
+    {}
+
+    virtual double value(const Point<spacedim> &p,
+                         unsigned int           component = 0) const
+    {
+      Tensor<1, spacedim> velocity;
+      if (spacedim == 3)
+        {
+          Tensor<1, spacedim> omega;
+          for (unsigned int i = 0; i < spacedim; ++i)
+            omega[i] = angular_velocity.value(p, i);
+
+          velocity = cross_product_3d(p, omega);
+        }
+
+      if (spacedim == 2)
+        {
+          double omega = angular_velocity.value(p, 0);
+
+          velocity[0] = -omega * p[1];
+          velocity[1] = omega * p[0];
+        }
+
+      return velocity[component];
+    }
+
+  private:
+    const Functions::ParsedFunction<spacedim> &angular_velocity;
+  };
+
+
+  template <int spacedim>
+  class SolidDisplacement : public Function<spacedim>
+  {
+  public:
+    SolidDisplacement(
+      const Functions::ParsedFunction<spacedim> &angular_velocity,
+      const double                               time_step)
+      : Function<spacedim>(spacedim)
+      , angular_velocity(angular_velocity)
+      , time_step(time_step)
+    {}
+
+    virtual double value(const Point<spacedim> &p,
+                         unsigned int           component = 0) const
+    {
+      Tensor<1, spacedim> displacement;
+
+      double dtheta = angular_velocity.value(p, 0) * time_step;
+
+      displacement[0] = std::cos(dtheta) * p[0] - std::sin(dtheta) * p[1];
+      displacement[1] = std::sin(dtheta) * p[0] + std::cos(dtheta) * p[1];
+
+      return displacement[component];
+    }
+
+    void set_time_step(const double new_time_step)
+    {
+      time_step = new_time_step;
+    }
+
+  private:
+    const Functions::ParsedFunction<spacedim> &angular_velocity;
+    double                                     time_step;
+  };
+
+  template <int dim, int spacedim = dim>
+  class StokesImmersedProblem
+  {
+  public:
+    StokesImmersedProblem(
+      const StokesImmersedProblemParameters<dim, spacedim> &par);
+
+    void run();
+
+  private:
+    void make_grid();
+    void setup_tracer_particles();
+    void setup_solid_particles();
+    void initial_setup();
+    void setup_dofs();
+    void assemble_stokes_system();
+    void assemble_nitche_restriction();
```

Comment:
This is just a random typo I wanted to point out:
```suggestion
    void assemble_nitsche_restriction();
```

Reply:
thanks!